charts/images.yaml

```diff
@@ -2,7 +2,7 @@ images:
 - name: etcd-backup-restore
   sourceRepository: github.com/gardener/etcd-backup-restore
   repository: eu.gcr.io/gardener-project/gardener/etcdbrctl
-  tag: "v0.15.0"
+  tag: "v0.15.1"
```

Comment:
I guess this should be the release that contains the [this PR](https://github.com/gardener/etcd-backup-restore/pull/465/).  I think it would be 1.16.something, so lets leave this as TODO for now.

/hold